rust/scripts/publish-release.sh

```diff
@@ -18,6 +18,14 @@ main() {
     local __release_file=$1
     local __release_name=$2
     local __release_tag="${CIRCLE_SHA1:0:16}"
+    local __release_tag_name="${CIRCLE_TAG:-default}"
+
+    if [[ ! -z ${__release_tag_name} && ${__release_tag_name} =~ ^v.*$ ]]; then
```

Comment:
Perhaps it makes sense to add a comment like "only publish if a release tag starting with `v` was pushed.